library/old_protocol/deserialize.cpp

```diff
@@ -99,9 +110,9 @@ int deserialize_lookup_response_body(dnet_node *n, const dnet_cmd &, data_pointe
 	return 0;
 }
 
-int deserialize_lookup_new_response_body(dnet_node *n, const dnet_cmd &, data_pointer &&message_buffer,
-                                         std::shared_ptr<n2_body> &out_deserialized) {
-	auto body = std::make_shared<lookup_response>();
+template <class Message>
+int deserialize_new(dnet_node *n, data_pointer &&message_buffer, std::shared_ptr<n2_body> &out_deserialized) {
```

Comment:
I think in the future we can rename it to `deserialize`, and for old handlers to make something alike (with templates), named `deserialize_deprecated`. Possibly, you can do it now, because after your changes `deserialize_lookup_response_body` looks inconsistent.

Reply:
Ok, I'm agree. But I didn't choose good names. I think we will often come across this code and choose good variant. Left as is.

Reply:
Ok.